rhtap/att-predicate-azure.sh

```diff
@@ -0,0 +1,33 @@
+#
+# Create attestation predicate for RHTAP Azure builds
+#
+# Useful references:
+# - https://slsa.dev/spec/v1.0/provenance
+# - https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#environment-variables
+# - https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#build-variables-devops-services
+#
+yq -o=json -I=0 << EOT
+---
+buildDefinition:
+  buildType: "https://redhat.com/rhtap/slsa-build-types/${CI_TYPE}-build/v1"
+  externalParameters: {}
+  internalParameters: {}
+  resolvedDependencies:
+    - uri: "git+${BUILD_REPOSITORY_URI}"
+      digest:
+        gitCommit: "${BUILD_SOURCEVERSION}"
+
+runDetails:
+  builder:
+    id: "${AGENT_ID}"
+
+  metadata:
+    invocationID: "${BUILD_BUILDURI}"
```

Comment:
Not sure what to do about it (if anything), and not a blocker for this PR, but I noticed we have `invocationID` here and for Jenkins, and `invocationId` for the other two.

Reply:
oof probably tells you where I started from :sweat_smile: 

I don't think anything relies on this, so I'll go ahead and make them consistent in a follow up PR. 

Good catch!

Reply:
https://github.com/redhat-appstudio/tssc-dev-multi-ci/pull/150